packages/services/api/src/modules/audit-logs/helpers.ts

```diff
@@ -0,0 +1,21 @@
+import { Injector } from 'graphql-modules';
+import { OrganizationManager } from '../organization/providers/organization-manager';
+import { AuditLogModel } from './providers/audit-logs-manager';
+
+export function resolveRecordAuditLog(event: AuditLogModel, injector: Injector) {
```

Comment:
Instead, this could be a method on the AuditLogsManager class, so we don't need this extra file here.